README.md

```diff
@@ -234,6 +235,24 @@ const customEmojis = [
 <Picker custom={customEmojis} />
 ```
 
+## Custom icons
+You can provide custom icons which will override the default icons. You can use svg (preferred) or png.
```

Comment:
Since we'll be using an iconfont, we should test to make sure that works too.

Reply:
Given that we now accept any react component, should we change/remove this line:

> You can use svg (preferred) or png.